src/test/java/EntityCalendarRecordSearchTest.java

```diff
@@ -0,0 +1,62 @@
+import base.BaseTest;
+import base.BaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+import utils.ProjectUtils;
+import utils.TestUtils;
+
+import java.util.List;
+
+import static utils.TestUtils.scroll;
+
+public class EntityCalendarRecordSearchTest extends BaseTest {
+
+    private void createBasicCalendarRecord(String str, String date){
+        TestUtils.scrollClick(getDriver(), findElement(By.xpath("//p[contains (text(), 'Calendar')]")));
+        findElement(By.xpath("//*[@class='card-icon']")).click();
+
+        findElement(By.id("string")).sendKeys(str);
+        findElement(By.id("date")).click();
+        findElement(By.id("date")).clear();
+        findElement(By.id("date")).sendKeys(date);
+
+        TestUtils.scrollClick(getDriver(), findElement(By.id("pa-entity-form-save-btn")));
+    }
+    @Test
+    public void testSearchCreatedRecord() {
```

Comment:
после 30 строчки добавьте пустую разделительную строчку
```suggestion
    }
    
    @Test
    public void testSearchCreatedRecord() {
```